test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -218,22 +218,20 @@ public void lraCoordinatorRecoveryTwoLRAs(@ArquillianResource @OperateOnDeployme
         restartContainer(LRA_COORDINATOR_CONTAINER_QUALIFIER);
 
         // Waits for a period of time longer than the timeout of the short LRA transaction
-        doWait((LRAListener.LRA_SHORT_TIMELIMIT + 1L) * 1000);
-
-        int sc = recover();
-
-        if (sc != 0) {
-            recover();
-        }
+        doWait(SHORT_TIMEOUT + 1000L);
 
         LRAStatus longStatus = getStatus(longLRA);
         LRAStatus shortStatus = getStatus(shortLRA);
 
         Assert.assertEquals("LRA with long timeout should still be active",
                 LRAStatus.Active.name(), longStatus.name());
+        if (shortStatus == LRAStatus.Cancelling) {
+            recover();
+            shortStatus = getStatus(shortLRA);
+        }
```

Comment:
@marcosgopen please can you clarify why this block is a better approach than the previous block: https://github.com/jbosstm/lra/pull/27/files#diff-1a03090f8ce8ff7635f6bc880a133ff1d5f01a56955be2da86c33199963bf879L221-L227 they look to be somewhat similar to me.

Reply:
After some recent changes (https://github.com/wildfly/wildfly/pull/18499) the recovery is now triggered by the server startup, so it should not be triggered manually.

Reply:
OK, that's clear then. I think I would say "should not need to be triggered manually" but that is clearer now to why it's different. Thank you